_locales/en/messages.json

```diff
@@ -1751,8 +1751,15 @@
     },
       "pidTuningRatesTip": {
         "message": "Play with the rates and see how those affect the stick curve"
+    },
+      "configurationMisc": {
+        "message": "Misc"
+    },
+      "configurationVesselName": {
+        "message": "Vessel name"
```

Comment:
'vessel' is a funny way to refer to something that flies, it is normally used for watercraft. I think 'craft' would be better here.